TDDUnitTests/UserControllerTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using AutoFixture;
+using TDDProject.Models;
+
+namespace TDDUnitTests
+{
+    public class UserControllerTests
+    {
+        Fixture _fixture;
+        private readonly UserController _controller;
+        public UserControllerTests()
+        {
+            _fixture = new Fixture();
+            _controller = new UserController();
+        }
+
+        [Fact]
+        public async Task GetByIdAsync_With_Correct_Request_Returns_Success_With_ResponseBody()
+        {
+            //Arrange
+            UserAutoFixture.GetUserFixture(_fixture);
+            User user = _fixture.Create<User>();
+
+            //Act
+            var response = await _controller.GetByIdAsync(user.UserId);
+            response.Should().BeOfType<OkObjectResult>();
+            var result = response as OkObjectResult;
```

Comment:
var result = response as OkObjectResult; is unecessary after  response.Should().BeOfType<OkObjectResult>();